### DISCUSSION TITLE
Storage: Video playback on mobile

### DISCUSSION THREAD

**Discussion by dommarr** [supabase/supabase#1739] [Questions] [answered]
Hi,

I'm storing mp4 files and having trouble getting them to play on mobile. Videos play as expected on desktop. 

For example, the signed url below plays on desktop, but does not play on mobile (iPhone X, Chrome + Safari). 

Any thoughts?

https://dnlvkovcawtkzuvpmmgr.supabase.co/storage/v1/object/sign/reviews/4b61adc2-6741-4331-b224-f0536a694646_1022?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1cmwiOiJyZXZpZXdzLzRiNjFhZGMyLTY3NDEtNDMzMS1iMjI0LWYwNTM2YTY5NDY0Nl8xMDIyIiwiaWF0IjoxNjIyMTU2Mzg2LCJleHAiOjE5Mzc1MTYzODZ9.HJTXO6PTTanwgItywnCBr-KSijL0Ar11JKIoXp_H4sY

**Comment by inian** [comment, marked as answer]
Hi @dommarr, I think it might be because of [this issue](https://github.com/supabase/storage-api/issues/20#issuecomment-843192722). Will work on this in the next couple of weeks.

**Reply by inian** [reply]
This has been fixed now @dommarr
